gbmodel/model_sqlalchemy.py

```diff
@@ -535,12 +551,61 @@ def get_sessions(self):
             lists.append(temp)
         return lists
 
+    def get_active_sessions(self):
+        """
+        Get a list of active capstone sessions
+        Input: self
+        Output: the list of currently active capstone sessions
+        """
+        # Calculate the start term and year of the sessions we expect to be active
+        currentDate = datetime.datetime.now()
+        month = int(currentDate.month)
+        if month in range(1, 3):
+            # Fall term of last year
+            start_term_1 = "Fall"
+            start_year_1 = currentDate.year - 1
+
+            # Winter term of current year
+            start_term_2 = "Winter"
+            start_year_2 = currentDate.year
+        else:
+            # Both terms will start in the same year
+            start_year_1 = currentDate.year
+            start_year_2 = currentDate.year
+
+            # Winter and Spring terms
+            if month in range(3, 6):
+                start_term_1 = "Winter"
+                start_term_2 = "Spring"
+            # Spring and Summer terms
+            elif month in range(6, 9):
+                start_term_1 = "Spring"
+                start_term_2 = "Summer"
+            # Summer and Fall terms
+            else:
+                start_term_1 = "Summer"
+                start_term_2 = "Fall"
+
+        # Query the db for active sessions using the start term and year information we calculated above
+        try:
+            # https://stackoverflow.com/questions/7942547/using-or-in-sqlalchemy
+            # Algorithm: SELECT * FROM CAPSTONE_SESSION WHERE
+            #               (start_term = start_term_1 AND start_year = start_year_1)
+            #                 OR
+            #               (start_term = start_term_2 AND start_year = start_year_2)
+            return capstone_session.query.filter(((capstone_session.start_year == start_year_1) &
+                                                  (capstone_session.start_term == start_term_1)) |
+                                                 ((capstone_session.start_year == start_year_2) &
+                                                  (capstone_session.start_term == start_term_2))).all()
+        except exc.SQLAlchemyError:
+            return None
+
     def check_dates(self, start, end):
         """
         Check if start and end dates are valid
         Input: start and end dates
         Output: Return 0 if valid, return 1 if start date is after the end date
-            Return 1 if either start or end date is empty
+                Return 1 if either start or end date is empty
```

Comment:
I noticed this while rebasing. Not sure if I needed to make that spacing change, but I wanted to